src/graphics/program-lib/chunks/reflectionCube.frag

```diff
@@ -1,7 +1,15 @@
 uniform samplerCube texture_cubeMap;
 uniform float material_reflectivity;
+#ifdef CLEARCOAT
+    uniform float material_clear_coat_reflectivity;
```

Comment:
What is the general naming convention in shaders? I would have expected this to be named material_clearCoatReflectivity, but I could be wrong.

Reply:
Yes, you're right @slimbuck, that is the convention.

Reply:
yes - I think you are right

Reply:
renamed - thanks!